### PR TITLE
Fix broken tests on Jenkins

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title

### CODE REVIEW

**Comment by s0undt3ch** [comment]
re-run full all

**Review by MKLeb — DISMISSED** [review]

**Comment by s0undt3ch** [comment]
re-run full all

**Comment by s0undt3ch** [comment]
re-run full all

**Comment by s0undt3ch** [comment]
re-run full all

**Comment by s0undt3ch** [comment]
re-run full all

**Comment by s0undt3ch** [comment]
re-run full all

File: tests/unit/modules/test_zcbuildout.py [hunk 1]
```diff
@@ -243,8 +243,16 @@ def test_get_bootstrap_content(self):
             "",
             buildout._get_bootstrap_content(os.path.join(self.tdir, "var", "tb", "1")),
         )
+
+        if (
+            salt.utils.platform.is_windows()
+            and os.environ.get("GITHUB_ACTIONS_PIPELINE", "0") == "0"
```
Comment (twangboy) [inline tests/unit/modules/test_zcbuildout.py:249]: Why are we checking for `GITHUB_ACTIONS_PIPELINE` here. Wouldn't we always want linebreaks to be `\r\n` on Windows?

**Review by twangboy — CHANGES_REQUESTED** [review]

File: tests/unit/modules/test_zcbuildout.py (hunk 1 above)
Comment (s0undt3ch) [reply tests/unit/modules/test_zcbuildout.py:249]: Under GH pipelines, we use `ssh` to run command on the VM, a windows build of `OpenSSH`.
Line breaks on windows, through `ssh` are `\n`...

**Review by twangboy — APPROVED** [review]

**Comment by s0undt3ch** [comment]
Extraordinary self merge to fix Jenkins tests.

**Pull request merged**